exercises/practice/clock/Clock.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS
+    Implement a clock that handles times without dates.
+
+.DESCRIPTION
+    Implement a clock that handles times without dates in 24 hours format.
+    You should be able to add and subtract minutes to it.
+    Two clocks that represent the same time should be equal to each other.
+    Note: Please try to implement the class and its method instead of using built-in module Datetime.
+
+.EXAMPLE
+    $clock1 = [Clock]::new(5,0)
+    $clock1.ToString()  => "05:00"
+
+    $clock2 = [Clock]::new(6,-120)
+    $clock2.Add(60).ToString() => "05:00"
```

Comment:
I think it could make sense to use `Return` as the other 2 prs instead of `=>`.

Reply:
All fixed!